include/ofi_list.h

```diff
@@ -388,11 +388,13 @@ struct slist_entry {
 struct slist {
 	struct slist_entry	*head;
 	struct slist_entry	*tail;
+	OFI_DBG_VAR(size_t, size);
```

Comment:
Because of how this macro works, we can't add the ';' at the end of the line.  Some compilers don't like that.